code/languages/C++/Friend_function/README.md

```diff
@@ -0,0 +1,73 @@
+## friend Function
+
+A friend function of a class is defined outside that class' scope but it has the right to access all private and protected members of the class. Even though the prototypes for friend functions appear in the class definition, friends are not member functions.
+A friend can be a function, function template, or member function, or a class or class template, in which case the entire class and all of its members are friends.
```

Comment:
Try to also cover the limitations of friend functions.

Reply:
Done!